architects/demo/sale_order_line.xml

```diff
@@ -0,0 +1,142 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="sale_order_line_1" model="sale.order.line">
+        <field name="remaining_hours">1.0</field>
```

Comment:
Shouldn't it be the `product_uom_qty`, from which the `remaining_hours` is deduced?

Reply:
But then you can remove this line...